package.json

```diff
@@ -65,7 +65,7 @@
     "mock-release": "node ./script/ci-release-build.js",
     "preinstall": "node -e 'process.exit(0)'",
     "publish-to-npm": "node ./script/publish-to-npm.js",
-    "precommit": "python script/run-clang-format.py -r -c atom/ chromium_src/ brightray/ && npm run lint:js && remark docs -qf || (echo \"Code not formatted correctly.\" && exit 1)",
+    "precommit": "python script/run-clang-format.py -r -c atom/ chromium_src/ brightray/ && node ./script/cpplint.js -c && npm run lint:js && remark docs -qf || (echo \"Code not formatted correctly.\" && exit 1)",
```

Comment:
This is getting long, can we put it in a `script/precommit.py` or something?

Reply:
I started to write one but couldn't think what else we might put into it, so I opted for inserting it here.

You're right though that it's pretty long. I'll followup with a `script/precommit` file